index.html

```diff
@@ -305,6 +305,10 @@ <h4 id=role-map-abstract><code>doc-abstract</code></h4>
         <a class="dpub-role-reference" href="#doc-abstract"><code>doc-abstract</code></a>
       </td>
     </tr>
+    <tr>
+      <th><a class="core-mapping" data-cite="core-aam#roleMappingComputedRole">Computed Role</a></th>
```

Comment:
@spectranaut This link references Core-AAM 1.2, which is not yet live. What should it be instead? ED link?

Reply:
NM... My mistake. Should be resolved now.